code/datums/controllers/process/gang.dm

```diff
@@ -44,14 +44,23 @@
 				gang.next_spray_paint_restock = timestring
 
 
-/datum/controller/process/gang_launder_money
+/datum/controller/process/gang_locker_tick
 	setup()
-		name = "Gang_Money_laundering"
+		name = "Gang Locker Tick"
 		schedule_interval = GANG_LAUNDER_DELAY
 	doWork()
 		for_by_tcl(locker, /obj/ganglocker)
 			if (!locker)
 				return
+			if (!locker.is_hiding)
+				var/should_hide = TRUE
```

Comment:
This var doesn't actually do anything because the loop returns if it finds a ganger

Reply:
good spot. brain farded